src/Types/index.ts

```diff
@@ -176,3 +194,53 @@ export type FilterProps = {
 export interface CallAPIProps extends AxiosRequestConfig {
   endpoint: string;
 }
+
+export type FileUploadTypes = {
+  uid: string;
+  lastModified: number;
+  lastModifiedDate: string;
+  name: string;
+  size: number;
+  type: string;
+  percent: number;
+  originFileObj: {
+    uid: string;
+  };
+};
+
+export interface CreateTalentTypes {
+  firstName: string;
+  lastName: string;
+  tanggalLahir: string;
+  domisili: string;
+  phone: string;
+  email: string;
+  divisi: string;
+  posisi: string;
+  kontrak: number;
+  salary: number;
+  skills: string[];
+  bahasa: string[];
+  github: string;
+  linkedin: string;
+  websitePortfolio: string;
+  status: "idle" | "on going";
+  isAvailableWFO: boolean;
+  cv: string;
+  tentangDiri: string;
+}
```

Comment:
kalau dari BE ga provide, kita bikin standard in bahasanya, kalau inggris, inggris terus, kalau indonesia indonesia semua